src/main/webapp/i18n/de/lti.json

```diff
@@ -13,9 +13,9 @@
                 }
             },
             "startExercise": "<strong>Herzlich Willkommen bei Artemis!</strong> Klicke auf den Button <i>Aufgabe starten</i>!",
+            "ltiSuccessLoginRequired": "LTI Übung erfolgreich gestartet. Bitte login als User {{user}}.",
```

Comment:
The second one is not a German sentence

Reply:
Can you maybe suggest a better way to word it?

Would "LTI Übung erfolgreich gestartet. Meldet dich als Benutzer {{user}} an." be better?

Reply:
"Bitte melde dich als Nutzer {{user}} an" (IMO User instead of Nutzer should be fine though)

Reply:
Thanks everyon, I changed the wording to what @simon-hng recommended